test/app.test.ts

```diff
@@ -139,3 +146,46 @@ describe('Import Zip', () => {
     });
   });
 });
+
+describe('Export Zip', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+
+    jest.spyOn(runner, 'runSingle').mockImplementation(async (task) => task.result);
+    jest.spyOn(runner, 'runSingleSequence').mockImplementation(async (tasks) => tasks[0].result);
+  });
+
+  it('Successfully export zip', async () => {
+    const app = await build({
+      taskManager,
+      runner,
+    });
+
+    jest.spyOn(taskManager, 'createGetTask').mockImplementation((member, itemId) => {
+      if (ITEM.id === itemId) return new MockTask(ITEM);
+      SUB_ITEMS.forEach((item) => {
+        if (item.id === itemId) return new MockTask(item);
+      });
+      return new MockTask(null);
+    });
+    const createGetChildrenTask = jest
+      .spyOn(taskManager, 'createGetChildrenTaskSequence')
+      .mockImplementation((member, itemId) => {
+        if (ITEM.id === itemId) return [new MockTask(SUB_ITEMS)];
+        else return [new MockTask([])];
+      });
+
+    const res = await app.inject({
+      method: 'GET',
+      url: `/zip-export/${ITEM.id}`,
+      headers: new FormData().getHeaders(),
+    });
+
+    expect(res.statusCode).toBe(StatusCodes.OK);
+    expect(res.headers['content-type']).toBe('application/octet-stream');
+    expect(res.headers['content-disposition']).toBe('filename=\"parent_public_item.zip\"');
```

Comment:
Where does this filename come from? Get it from your mock data instead of hardcoding it.